docs/getting-started-guides/README.md

```diff
@@ -20,13 +20,99 @@ certainly want the docs that go with that version.</h1>
 <!-- END STRIP_FOR_RELEASE -->
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
-If you are not sure what OSes and infrastructure is supported, the table below lists all the combinations which have
-been tested recently.
 
-For the easiest "kick the tires" experience, please try the [local docker](docker.md) guide.
+# Creating a Kubernetes Cluster
 
-If you are considering contributing a new guide, please read the
-[guidelines](../../docs/devel/writing-a-getting-started-guide.md).
+Kubernetes can run on a range of platforms, from your laptop, to VMs on a cloud provider, to rack of
+bare metal servers.  The effort required to setup a cluster varies from running a single command to
+crafting your own customized cluster.  We'll guide you in picking a solution that fits for your needs.
+
+## Picking the Right Solution
+
+If you just want to "kick the tires" on Kubernetes, we recommend the [local Docker-based](docker.md) solution.  
+
+The local Docker-based solution is one of several [Local cluster](#local-solutions) solutions
+that are quick to setup, but are limited to running on one machine.
+
+When you are ready to scale-up to more machines and higher availability, a [Hosted](#hosted-solutions)
+solution is the easiest to create and maintain.
+
+[Turn-key cloud solutions](#turn-key-cloud-solutions) require only a few commands to create
+and cover a wider range of cloud providers.
+
+[Custom solutions](#custom-solutions) require more effort to setup but cover and even 
+they vary from step-by-step instructions to general advice for setting up
+a kubernetes cluster from scratch. 
+
+### Local-machine Solutions
+These local-machine solutions create a single cluster with one or more kubernetes nodes on a single
```

Comment:
If you were referring to the ones listed in the next paragraph, then get rid of "Other automated local-machine solutions are:" and just move the bullet list up to merge with this paragraph